components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -224,6 +236,7 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
         /// </summary>
         /// <param name="state"></param>
         private void BatchTriggerIntervalTimer_Elapsed(object state) {
+            _batchTriggerIntervalTimer.Change(Timeout.Infinite, Timeout.Infinite);
```

Comment:
Should not be needed if you only trigger once

Reply:
done